reference/fleet/providers.md

```diff
@@ -53,34 +72,34 @@ Context providers give the current context of the running {{agent}}, for example
 
 They can only provide a single key-value mapping. Think of them as singletons; an update of a key-value mapping results in a re-evaluation of the entire configuration. These providers are normally very static, but not required. A value can change which results in re-evaluation.
 
-Context providers use the Elastic Common Schema (ECS) naming to ensure consistency and understanding throughout documentation and projects.
+Context providers use the {{product.ecs}} naming to ensure consistency and understanding throughout documentation and projects.
 
 {{agent}} supports the following context providers:
 
-* [Local](/reference/fleet/local-provider.md)
-* [Agent Provider](/reference/fleet/agent-provider.md)
-* [Host Provider](/reference/fleet/host-provider.md)
-* [Env Provider](/reference/fleet/env-provider.md)
-* [Kubernetes Secrets Provider](/reference/fleet/kubernetes_secrets-provider.md)
-* [Kubernetes Leader Election Provider](/reference/fleet/kubernetes_leaderelection-provider.md)
+* [Local provider](/reference/fleet/local-provider.md)
+* [Agent provider](/reference/fleet/agent-provider.md)
+* [Host provider](/reference/fleet/host-provider.md)
+* [Env provider](/reference/fleet/env-provider.md)
+* [Kubernetes Secrets provider](/reference/fleet/kubernetes_secrets-provider.md)
+* [Kubernetes LeaderElection provider](/reference/fleet/kubernetes_leaderelection-provider.md)
 
 
-### Dynamic Providers [dynamic-providers]
+### Dynamic providers [dynamic-providers]
 
 Dynamic providers give an array of multiple key-value mappings. Each key-value mapping is combined with the previous context provider’s key and value mapping which provides a new unique mapping that is used to generate a configuration.
 
 {{agent}} supports the following context providers:
 
-* [Local Dynamic Provider](/reference/fleet/local-dynamic-provider.md)
-* [Docker Provider](/reference/fleet/docker-provider.md)
-* [Kubernetes Provider](/reference/fleet/kubernetes-provider.md)
+* [Local dynamic provider](/reference/fleet/local-dynamic-provider.md)
+* [Docker provider](/reference/fleet/docker-provider.md)
+* [Kubernetes provider](/reference/fleet/kubernetes-provider.md)
 
 
-### Disabling Providers By Default [disable-providers-by-default]
+### Disabling providers by default [disable-providers-by-default]
 
 All registered providers are disabled by default until they are referenced in a policy.
```

Comment:
I wouldn't use disabled here. I would say they are not running, not disabled. They are ran when they are referenced.